app/uk/gov/hmrc/fileupload/controllers/envelope.scala

```diff
@@ -60,45 +60,40 @@ object EnvelopeReport {
 case class CreateEnvelopeRequest(callbackUrl: Option[String] = None,
                                  expiryDate: Option[DateTime] = None,
                                  metadata: Option[JsObject] = None,
-                                 constraints: Option[EnvelopeConstraintsUserO] = None)
+                                 constraints: Option[EnvelopeConstraintsUserO] = Some(EnvelopeConstraintsUserO()))
```

Comment:
Konrad showed me this nice way to do it in the serialization layer:
val fooReader: Reads[String] = new Reads[String] {
  val REGEX = """ YOUR REGEX for MB etc. """.r
  def reads(json: JsValue) = {
    json match {
      case JsString(REGEX(s)) => JsSuccess(s)
      case _ => JsError(s"unable to parse: $json as size constraint")
    }
  }
}